ntoskrnl/ke/i386/kiinit.c

```diff
@@ -536,7 +538,7 @@ KiInitializeKernel(IN PKPROCESS InitProcess,
     else
     {
         /* FIXME */
-        DPRINT1("Starting CPU#%u - you are brave\n", Number);
```

Comment:
https://github.com/reactos/reactos/pull/6014/commits/223ad271e314f868e6fc1e30a00b2d642c66f7c7#r1398588235

Same thing here, Revert this

Reply:
I do like Serges solution, even if it has "less personality" as you would call it.
Two people here now already with the same thing in mind.
My 2 cents: Strip the bloat right now, before it's 10 years old. After all, what's the benefit of telling the user he is brave? He will file a ticket if it crashes in both cases: whether he is brave, or whether he is a "pussy".

Reply:
I'm not commenting on his text changes, he's denoting information i need to see when people try to test this. 

Don't make SMP bring up more difficult I'm already in way over my head.

Reply:
Maybe you missed that Serge did not remove that line, but just shortened the string to
DPRINT("Starting CPU %d\n", Number);

What is the added value that the user is brave? We do know that this code is fresh and potentially unfinished.